File: server/src/models/card.ts
import * as mongoose from 'mongoose';

export interface Card extends mongoose.Document {
    id: number;
    name: string;
    hp: number;
    superSkill: string;
    ignore: string[];
    damage: number;
    image: string;
}

export const CARD_SCHEMA = {
    name: {
        type: String,
        required: true
    },
    hp: {
        type: Number,
        required: true
    },
    image: {
        type: String,
    },
    superSkill: {
        type: String,
        required: true
    },
    ignore: {
        type: [String],
    },
    createAttack: {
        type: [String],
    },
};

File: go-server/domain/card.go
package domain

type Card struct {
	Class    string      `json:"class"`
	Damage   string      `json:"damage"`
	Effects  interface{} `json:"effects"`
	Hp       string      `json:"hp"`
	ID       string      `json:"id"`
	Image    string      `json:"image"`
	ManaCost string      `json:"manaCost"`
	Name     string      `json:"name"`
	Skills   []string    `json:"skills"`
	SurName  string      `json:"surName"`
}
